build/mips-simd-check.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#**********************************************************************************
+#    This script is using in build/arch.mk for mips to detect the simd instructions:
+#    mmi, msa (maybe more in the future).
+#
+#   --usage:
+#             ./mips-simd-check.sh $(CC) mmi
+#         or  ./mips-simd-check.sh $(CC) msa
+#
+# date:  10/17/2019 Created
+#**********************************************************************************
+
+TMPC=$(mktemp test.XXXXXX.c)
```

Comment:
recommended change the file name to "test-mmi.XXXXXX.c"/ "tmp-mmi.XXXXXX.c"/"tmp.XXXXXX.c" and so on.

Reply:
ok, i will rename to tmp.XXXXXX.c.